Sources/swiftML/Helpers.swift

```diff
@@ -0,0 +1,70 @@
+import TensorFlow
+
+extension Tensor where Scalar: TensorFlowFloatingPoint {
+    /// The (Moore-Penrose) pseudoinverse of a matrix using its singular-value decomposition (SVD).
+    public var pseudoinverse: Tensor {
+        let svd = Raw.svd(self)
+        let diag = Raw.diag(diagonal: svd.s)
+        return matmul(matmul(svd.v, Raw.matrixInverse(diag)), svd.u.transposed())
+    }
+}
+
+/// Returns sign-corrected versions of the left and right singular vectors from
+/// single value decomposition (SVD).
+///
+/// - Parameters:
+///   - u: The tensor containing the left singular vectors for each matrix.
+///   - v: The tensor containing the right singular vectors for each matrix.
+/// - Returns: The sign corrected versions of `u` and `v` to ensure
+///   deterministic output.
+public func svdFlip<Scalar: TensorFlowFloatingPoint>(
+    u: Tensor<Scalar>,
+    v: Tensor<Scalar>,
+    uBasedDecision: Bool = true
```

Comment:
The purpose/naming of the parameter `uBasedDecision: Bool = true` isn't super clear to me. Could you please explain (or share a link) what it represents?

Let's work on a parameter doc comment and maybe a new name.